src/components/ButtonComponents/NumberButtons/Numbers.js

```diff
@@ -1,16 +1,31 @@
 import React from "react";
+import NumberButton from "./NumberButton";
 
 //import any components needed
+import {data} from "../../../data"
 
 //Import your array data to from the provided data file
-
+// console.log(numbers,"HERRRREEE");
```

Comment:
consider clearing redundant console.logs or any other block of code you do not need before pushing your commits